src/common/settings.js

```diff
@@ -508,6 +508,13 @@ const character_settings = {
 
     // effects
 
+    "effects-exhaustion": {
+        "title": "Effect: Exhaustion",
+        "description": "Applies 2024 rules for exhaustion. This condition is cumulative and effects all D20 roles applying a -2 * exhaustion level modifier to the roll.",
+        "type": "bool",
+        "default": false
```

Comment:
I think this is the kind of effect that deserves having a default of `true`, since it won't have any effect if the character has no exhaustion levels.

Reply:
I would turn it true only if it was version 2024 character